documentation/library_guide/kernel_templates/single_pass_scan.rst

```diff
@@ -0,0 +1,208 @@
+
+.. code:: cpp
+
+   // defined in <oneapi/dpl/experimental/kernel_templates>
+
+   namespace oneapi::dpl::experimental::kt::gpu {
+
+   template <typename InIterator, typename OutIterator, typename BinaryOp>
+   sycl::event
+   inclusive_scan (sycl::queue q, InIterator in_begin, InIterator in_end, OutIterator out_begin,
+                   BinaryOp binary_op, KernelParam param); // (1)
+
+   template <typename InRng, typename OutRng, typename BinaryOp>
+   sycl::event
+   inclusive_scan (sycl::queue q, InRng in_rng, OutRng out_rng, BinaryOp binary_op,
+                   KernelParam param) // (2)
+
+   }
+
+
+Parameters
+----------
+
++-----------------------------------------------+---------------------------------------------------------------------+
+| Name                                          | Description                                                         |
++===============================================+=====================================================================+
+|  ``q``                                        | SYCL* queue to submit the kernels to.                               |
++-----------------------------------------------+---------------------------------------------------------------------+
+|                                               | The sequences of elements to apply the algorithm to.                |
+|  - ``in_begin``, ``in_end``, ``out_begin`` (1)| Supported sequence types:                                           |
+|  - ``in_rng``, ``out_rng`` (2)                |                                                                     |
+|                                               | - ``sycl::buffer`` (2),                                             |
+|                                               | - :ref:`oneapi::dpl::experimental::ranges::views::all               |
+|                                               |   <viewable-ranges>` (2),                                           |
+|                                               | - :ref:`oneapi::dpl::experimental::ranges::views::subrange          |
+|                                               |   <viewable-ranges>` (2),                                           |
+|                                               | - :ref:`USM pointers <use-usm>` (1),                                |
+|                                               | - :ref:`oneapi::dpl::begin and oneapi::dpl::end                     |
+|                                               |   <use-buffer-wrappers>` (1).                                       |
+|                                               |                                                                     |
++-----------------------------------------------+---------------------------------------------------------------------+
+|  ``binary_op``                                | A function object that will applied to the elements of the input    |
```

Comment:
Please change to:  A function object that is applied to the elements of the input.